flowzone.yml

```diff
@@ -232,6 +232,11 @@ on:
         type: string
         required: false
         default: ""
+      custom_website_build:
+        description: "Runs a custom command to build the website. The command needs to be in a npm script stored in `npm run deploy`. Skipped if empty or not true."
+        type: boolean
+        required: false
+        default: false
```

Comment:
I would much rather a toggle like `docusaurus_website` boolean and we could default to `true`.

Reply:
After reading your other comments, I get what you mean. Yeah. Let's do that. Cool